projects/demo/src/modules/services/destroy/destroy.template.html

```diff
@@ -0,0 +1,23 @@
+<tui-doc-page
+    header="DestroyService"
+    package="CDK"
+    path="cdk/services/destroy.service.ts"
+>
+    <ng-template pageTab>
+        <p i18n>Service to destroy another services</p>
```

Comment:
Check a description here: https://github.com/TinkoffCreditSystems/taiga-ui/wiki/TuiDestroyService

Reply:
I think it would be better to add one example of usage here